apostrophecms/apostrophe-documentation
# Add  APOS_BASE_URL documentation

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*

This PR adds the `APOS_BASE_URL` environment variable to several areas in the documentation. Closes PRO-3776 for the a2 documentation.

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

1) Build the documentation
2) Navigate to `/howtos/facebook.html#absolute-urls-in-apostrophe`,  `/reference/core-server.html#baseurl`, `/reference/modules/apostrophe-express.html`, and `/reference/modules/apostrophe-pages/#getbaseurl-req-api`

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:

- [ X] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
